Actions/ReadSettings/ReadSettings.ps1

```diff
@@ -54,6 +54,11 @@ if ($settings.versioningstrategy -ne -1) {
     }
 }
 
+if ($settings.buildEnvironmentName -and -not $settings.useCompilerFolder) {
+    Write-Host "Using an online environment for testing, use CompilerFolder to build the app"
+    $settings.useCompilerFolder = $true
```

Comment:
Why defaulting to compiler folder? The BC online environment is used for running the tests, right? Compiling the apps can still happen in a BC container.

Reply:
Why would you?
Containers are much slower than CompilerFolders and since you are running tests on an online environment, you will not need the container ever.
Also - if you switch to use an online environment because of dependencies or functionalities, which cannot be tested on a container - it will fail if you try to.
I cannot see any reason for building using a container and running tests in an online environment.

Reply:
For one, there could be overrides that do some additional setup before building the apps and test apps.

Switching like that, under the hood, is a breaking change. It must be documented at least.

Reply:
IMO - compiling with containers when using an online environment for testing will at best just be 4-5 times slower and at worst will fail slowly. But I agree that this should be more clearly documented under the setting - when you set this - you also do...

BTW - In the future (after getting rid of BcContainerHelper) - I don't think we should support building in containers at all.